src/A0_initialization.py

```diff
@@ -248,21 +249,28 @@ def process_user_arguments(
     parser = create_parser()
     args = vars(parser.parse_args())
 
-    # Give priority from kwargs over command line arguments
+    # Give priority from user input kwargs over command line arguments
+    # However the command line arguments have priority over default kwargs
     if path_input_folder is None:
```

Comment:
> if path_input_folder is None

Wouldn´t it be easier to get all items in DEFAULT_MAIN_KWARGS and then loop over them, sth like:

```
for  item in DEFAULT_MAIN_KWARGS:
    if item  is None:
        item ) args.get(item, DEFAULT_MAIN_KWARGS [item]
```

Reply:
(With "item" as the provided/not provided command line commands)

Reply:
`item` as you defined it here will never be `None` then. The user may or may not provide arguments to the main function and may or may not provide arguments to the command line at last resort, there are the default values from `DEFAULT_MAIN_KWARGS`